lua/core/telescope.lua

```diff
@@ -32,7 +32,9 @@ function M.config()
         vertical = { mirror = false },
       },
       file_sorter = require("telescope.sorters").get_fzy_sorter,
-      file_ignore_patterns = {},
+      file_ignore_patterns = {
+        ".git/",
```

Comment:
can we use some custom patterns here?

Reply:
Do you mean the config option should be exposed, so the user could set the ignore patterns in the config.lua?

Reply:
yeah that would be awesome